cli/src/semdep/parsers/package_lock.py

```diff
@@ -22,6 +23,96 @@
 logger = getLogger(__name__)
 
 
+def parse_packages_field(deps: Dict[str, JSON]) -> List[FoundDependency]:
+    deps_json = deps.get("packages")
+    if deps_json is None:
+        logger.debug("Found package-lock with no 'packages'")
+        return []
+
+    deps = deps_json.as_dict()
+    try:
+        manifest_deps = set(deps[""].as_dict()["dependencies"].as_dict().keys())
```

Comment:
the value you're looking for here typically has an empty string key?

Reply:
yes, check the example package-lock.json